Src/java/cqf-fhir-npm/src/main/java/org/cqframework/fhir/npm/LibraryLoader.java

```diff
@@ -40,7 +39,7 @@ public Library readLibrary(InputStream stream) throws FHIRFormatError, IOExcepti
             VersionConvertor_40_50 versionConvertor_40_50 = new VersionConvertor_40_50(new BaseAdvisor_40_50());
             return (Library) versionConvertor_40_50.convertResource(res);
         } else if (VersionUtilities.isR5Ver(version)) {
-            return (Library) new JsonParser().parse(stream);
+            return (Library) new org.hl7.fhir.r5.formats.JsonParser().parse(stream);
```

Comment:
nit: is it intenional?

Reply:
Yes. This is needed to avoid `org.objectweb.asm.ClassTooLargeException: Class too large: org/hl7/fhir/r5/formats/JsonParser`. See this [PR](https://github.com/hapifhir/org.hl7.fhir.core/pull/714) for more information on why that is required.